platforms/intellij/compose/playground/src/jvmTest/kotlin/foundry/intellij/compose/playground/MarkdownPlaygroundTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2024 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package foundry.intellij.compose.playground
+
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onRoot
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.runDesktopComposeUiTest
+import com.github.takahirom.roborazzi.RoborazziOptions
+import io.github.takahirom.roborazzi.captureRoboImage
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+class MarkdownPlaygroundKtTest {
+  @OptIn(ExperimentalTestApi::class)
+  @Test
+  fun test() = runTest {
```

Comment:
Bonus point. This test will test a light mode by default. If you want to test a dark mode, you can also try `@Config(qualifiers = "+night")` in another test.

Reply:
Do you know what dependency I need for this Config? I wasn't able to run the Roborazzi test due to a missing dependency, even though I have `robolectric` and `androidx-testing-core`. Is this annotation mobile specific?